### PR TITLE
Add missing expand/collapse indicators to assessment steps

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** Add missing expand/collapse arrows to assessment steps to provide better visual indication that these can be expanded/collapsed.

JIRA: Related to [EDUCATOR-5374](https://openedx.atlassian.net/browse/EDUCATOR-5374)

**What changed?**

- Add missing arrow to `Learn to Assess Responses` step when in-progress
- Add missing arrow to `Top Responses` step when it can be expanded
- Updated placeholder (loading) step titles to match loaded values

**Developer Checklist**

- [x] Reviewed the [release process](./release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

**Testing Instructions**

1. Create an ORA with at least `Learn to Assess Responses` step and `Show top responses` enabled
2. Submit a response
3. For each step in the process, verify that the `Learn to Assess Responses` step shows an expand/collapse arrow when appropriate (when clicking is able to expand/collapse)
4. For each step in the process, verify that the `Top Responses` section shows an expand/collapse arrow when appropriate (when clicking is able to expand/collapse)

**Before**
Screen Shot 2020-12-11 at 18 42 50

**After**
Screen Shot 2020-12-11 at 17 49 44

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [ ] I've tested the new functionality

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Comment by jansenk** [comment]
Visibly looks good but haven't tested locally

**Review by jansenk — APPROVED** [review]
No comments, looks great
